pkg/session_recording/player.jsx

```diff
@@ -591,6 +598,18 @@
             this.setState({ title: _("Player") + ": " + title });
         }
 
+        _transform(width, height) {
+            var relation = Math.min(
+              this.state.container_width / this.state.term.element.offsetWidth,
+              290 / this.state.term.element.offsetHeight
```

Comment:
What is this `290`? Could you please use a constant with a name which would give a clue instead?